src/utils.ts

```diff
@@ -44,7 +44,10 @@ export const getListNumber = (blockId: string, blockMap: BlockMapType) => {
 };
 
 export const toNotionImageUrl = (url: string) => {
-  return `https://notion.so${
+  const notionImageUrl = `https://www.notion.so${
```

Comment:
`notion.so/*` redirects to `www.notion.so/*` so it's useful to cut out the redirect